helical/models/geneformer/fine_tuning_model.py

```diff
@@ -0,0 +1,290 @@
+from typing import Literal, Optional
+from helical.models.base_models import HelicalBaseFineTuningHead, HelicalBaseFineTuningModel, HelicalRNAModel
+from helical.models.fine_tune.fine_tuning_heads import ClassificationHead
+from sklearn.metrics import accuracy_score
+import torch
+from torch import optim
+from torch.nn.modules import loss
+from helical.models.geneformer.geneformer_utils import gen_attention_mask, get_model_input_size, pad_tensor_list
+from datasets import Dataset
+import logging
+from tqdm import trange
+from transformers import get_scheduler
+
+
+logger = logging.getLogger(__name__)
+
+class GeneformerFineTuningModel(HelicalBaseFineTuningModel):
+    """GeneformerFineTuningModel
+    Fine-tuning model for the Geneformer model.
+    
+    Parameters
+    ----------
+    geneformer_model : Geneformer
+        The initialised Geneformer model to fine-tune.
+    fine_tuning_head : HelicalBaseFineTuningHead
+        The fine-tuning head that is appended to the model.
```

Comment:
Can you please add the options here? just to make sure we know what we can use as head?